### DISCUSSION TITLE
Feature tests response code 302 on authenticated routes

### DISCUSSION THREAD

**Discussion by KuMoNKo** [filamentphp/filament#463] [Help] [answered]
Hi!,
I seem to be unable to test any route where I have to be authenticated. I know I am authenticated in the test, but the response code is always 302. 
```
public function testAuthURL() {
        $user = User::factory()->create();
        $this->be($user);

        $this->assertAuthenticatedAs($user); //This one passes.

        $this->get(route('filament.account'))
        ->assertSuccessful();                //This line gives error 302
}

```
I checked the route is correct and tried using the url also. In the browser the route returns 200. Any Idea?

**Comment by danharrin** [comment, marked as answer]
Can you try this method in your test case?

```php
protected function getEnvironmentSetUp($app)
{
    $app['auth']->shouldUse('filament');
}
```

**Reply by KuMoNKo** [reply]
Ok, that's not the solution, but thanks to it I discovered that I forgot to modify the Laravel's original testcase to extend form Orchestra Testbench.
Thank you.
